### DISCUSSION TITLE
[BUG] Invoke-PnPSiteTemplate Erroring Out

### DISCUSSION THREAD

**Discussion by katjones23** [pnp/powershell#2854] [Q&A]
We have a PowerShell script to automatically provision extranet sites.  On February 3rd, we had a requested site that was completed successfully.  On the 23rd, we had a couple of requests come in that were not completed.  The issue is with Invoke-PnPSiteTemplate; in troubleshooting, we have found that it specifically is failing when trying to apply lists from the template.  We made no changes to the script or updated any modules in between the 3rd and the 23rd.  In another environment, we tested with downloading the latest stable releases of PnP.Powershell and Microsoft.Online.SharePoint.PowerShell, as well as older versions, and the error still occurred.  

The template is generated from an existing model site using Get-PnPSiteTemplate.  When we download a template of that same site with the parameter "-ExcludeHandlers Lists", the template is successfully applied.  When I use the site from the 3rd as a template, the same error occurs.

The error thrown:
<img width="356" alt="Invoke-PnPSiteTemplate_Error" src="https://user-images.githubusercontent.com/53064219/221662458-030166bc-2bdf-4022-bb8c-8498b60fabac.png">

Installed modules:
<img width="469" alt="2 27 23InstalledModules" src="https://user-images.githubusercontent.com/53064219/221663151-ed65d081-3ec1-47cd-a2a5-865c35bf7e33.png">

**Comment by fpcwashington** [comment]
Same error using PnP.Powershell v1.12.0.  The command just stopped working importing *.pnp site templates.
